warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__trip_id_alignment.sql

```diff
@@ -0,0 +1,56 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__rt_feed_guideline_index') }}
+),
+
+fct_daily_rt_feed_files AS (
+    SELECT * FROM {{ ref('fct_daily_rt_feed_files') }}
+),
+
+fct_daily_rt_feed_validation_notices AS (
+    SELECT * FROM {{ ref('fct_daily_rt_feed_validation_notices') }}
+),
+
+count_files AS (
+    SELECT
+        base64_url,
+        date,
+        SUM(parse_success_file_count) AS rt_files
+    FROM fct_daily_rt_feed_files
+    GROUP BY 1, 2
+),
+
+critical_notices AS (
+    SELECT
+        date,
+        base64_url,
+        SUM(total_notices) AS errors,
+    FROM fct_daily_rt_feed_validation_notices
+    -- Description for code E003:
+    ---- "All trip_ids provided in the GTFS-rt feed must exist in the GTFS data, unless the schedule_relationship is ADDED"
```

Comment:
This is a way more efficient way to check this, great point (relative to manually making a separate incremental table and checking)